src/lib.rs

```diff
@@ -33,8 +35,7 @@ mod tests {
 
     #[test]
     fn encode_test_1(){
-        hex_to_base64("test");
-        // assert!("SSdtIGtpbGxpbmcgeW91ciBicmFpbiBsaWtlIGEgcG9pc29ub3VzIG11c2hyb29t", );
+        assert_eq!(hex_to_base64("test"), "dGVzdAo=");
```

Comment:
Command line `base64` command shows that this should be the expected output, but it's currently failing.